content/helloworld-fromcftok8s.md

```diff
@@ -235,7 +237,7 @@ Close but no cigar yet - I don't have a an actual domain name for my app like I
 
 To get things lifted up to the level of domains and hostnames as opposed to IP addresses and ports, I need yet another Kubernetes resource, called Ingress. An Ingress basically specifies how incoming requests should be routed to services. For example, if your app consists of multiple services, you can use an ingress to make sure requests to http://my-app.my-domain/service1 go to service 1, and http://my-app.my-domain/service2 go to service 2.  
 
-For the very simple hello world example we're discussing here, it is again really simple to define an Ingress that does the job of assigning a domain name to my app:
+For the simple hello world example we're discussing here, it is easy to define an Ingress that does the job of assigning a domain name to my app:
```

Comment:
Easy still isn't the right word. Maybe "a single file"? Not sure. But there's a good article I read recently pointing out that "easy" is likely not easy for beginners